src/editor/Objects/LatheController.tsx

```diff
@@ -0,0 +1,36 @@
+import { useRef, useState } from 'react';
+import * as THREE from 'three';
+import { ThreeElements, useFrame } from '@react-three/fiber';
+
+function LatheController(props: ThreeElements['mesh']) {
+  const ref = useRef<THREE.Mesh>(null!);
+  const [hovered, hover] = useState(false);
+  const [clicked, click] = useState(false);
+
+  const points = [];
+  for (let i = 0; i < 10; i++) {
+    points.push(new THREE.Vector2(Math.sin(i * 0.2) * 1.5 + 1, (i - 5) * 0.4));
+  }
```

Comment:
to samo jako wyzej. na razie niech to wejdzie, ale bedzie trzeba otworzyc taska zeby przemyslec UI na tworzenie bryl z punktow